src/index.js

```diff
@@ -89,12 +114,26 @@ app.get('/poll', (req, res) => {
   res.send({log: rollLog})
 })
 
+app.post('/clearLog', (req, res) => {
```

Comment:
it might make sense for this to be `delete`, what do you think?

Reply:
I agree

Reply:
as in `app.delete`  and the FE request method (line 70) would need to be DELETE too

also to semantically restful we should probably change the path from `/clearLog` to `/log`